cloudinit/util.py

```diff
@@ -68,17 +68,28 @@
                    ['running-in-container'],
                    ['lxc-is-container'])
 
-PROC_CMDLINE = None
+_CACHED_RESPONSES = {}
 
-_LSB_RELEASE = {}
 
+try:
```

Comment:
Imports should go above the global definitions.

Reply:
why doesn't pycodestyle / pylint complain about that?

Reply:
Once an import is in a `try/except`, there's probably something \~complicated~ happening so a rule which suggested moving those to the top of the file would _probably_ give false positives.

Or, perhaps, it's simply that no-one has implemented such a check.